hooks/useCheckBiometrics.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from "react";
+import * as LocalAuthentication from "expo-local-authentication";
+
+// Checks if the device we're on allows biometric auth
+export const useCheckBiometrics = () => {
+  const [isBiometricsSupported, setIsBiometricsSupported] = useState(false);
+
+  const checkBiometrics = async () => {
+    const compatible = await LocalAuthentication.hasHardwareAsync();
+    const enrolled = await LocalAuthentication.isEnrolledAsync();
+    setIsBiometricsSupported(compatible && enrolled);
+  };
+
+  useEffect(() => {
+    checkBiometrics();
+  }, []);
```

Comment:
if we have eslint  exhaustic-deps rule enabled it will complain about the missing dep, in the case we should add it on the deps array and use a usecallback on the function declaration

Reply:
Indeed! I've just updated useCheckBiometrics in this PR. Since I have changes in useNetworkDetection on my other PR, I'll improve it there